src/player.ts

```diff
@@ -1716,6 +1701,46 @@ export default class Player extends FakeEventTarget {
     return this._engine.getDrmInfo();
   }
 
+  private _getSubtitleStyleSheet(): CSSStyleSheet {
+    let element = Utils.Dom.getElementBySelector(`.${this._playerId}.${SUBTITLES_STYLE_CLASS_NAME}`);
+    if (!element) {
+      element = Utils.Dom.createElement('style');
+      Utils.Dom.addClassName(element, this._playerId);
+      Utils.Dom.addClassName(element, SUBTITLES_STYLE_CLASS_NAME);
+      Utils.Dom.appendChild(document.head, element);
+    }
+    return element.sheet;
+  }
+
+  private _resetCustomSubtitleStyles(): void {
+    const sheet = this._getSubtitleStyleSheet();
+    while (sheet.cssRules.length) {
+      sheet.deleteRule(0);
+    }
+  }
+
+  private _applyCustomSubtitleStyles(): void {
+    try {
+      const sheet = this._getSubtitleStyleSheet();
+
+      if (this._config.text.useNativeTextTrack && !this._config.text.useShakaTextTrackDisplay) {
+        sheet.insertRule(`#${this._playerId} video.${ENGINE_CLASS_NAME}::-webkit-media-text-track-display { text-align: ${this._textStyle.textAlign}!important; }`, 0);
```

Comment:
does the webkit-X prefix exist for every browser or can also it be media-text-track display or moz-text-track-display or something like that ?